### DISCUSSION TITLE
Ability to add the site previously established with Vito deploy

### DISCUSSION THREAD

**Discussion by furkankurt** [vitodeploy/vito#191] [Ideas]
I want to add the server hosting my Laravel project, which I deployed with Vito on my local machine, to my remote Vito Deploy site. I think that seems impossible right now.

**Comment by saeedvaziry** [comment]
There is no automation for this here you can manually migrate a local hosted VitoDeploy to a Remote hosted VitoDeploy or vice versa:

For example you want to move your local Vito to a Remote server

1. Make a backup of both `storage` folders (local and server)
2. Delete the `storage` folder on the server and copy the `storage` folder of your local Vito to your server
3. Make sure the `.env` file in the server has the same `APP_KEY` as your local
